components/home/constants.js

```diff
@@ -3,6 +3,7 @@ export const NAME = 'home';
 
 /** @constant */
 export const INITIAL_STATE = {
+    fakeUserId: ' ',
```

Comment:
delete this, given that you are using the userId in `reducer.js`

Reply:
Alright.